### PR TITLE
MT-878 Removed padding, margin and border LEFT for summary-detail style

### PR DESCRIPTION
**Pull request by miloszhygen**

### CODE REVIEW

File: src/scss/components/_summary-details.scss [hunk 1]
```diff
@@ -44,10 +44,6 @@ details {
     }
     .summary-wrapper {
       @include xp-html-area;
-
-      margin-left: var(--detail-margin);
```
Comment (lha047) [inline src/scss/components/_summary-details.scss:48]: Should this be for all summary details?

**Review by lha047 — CHANGES_REQUESTED** [review]
Just wonder if this should be removed from all summery details

**Review by lha047 — CHANGES_REQUESTED** [review]

**Review by lha047 — CHANGES_REQUESTED** [review]
Screenshot 2022-03-01 at 13 34 24

Den brune linja under oppsummeringen skal ikke være der på den uten linjer, padding og marg :P

**Comment by miloszhygen** [comment]
fixed @lha047 That was the conflict when rebasing.

**Review by lha047 — APPROVED** [review]
All good :D Good job

**Pull request merged**